### PR TITLE
adding program to add numbers

### PR DESCRIPTION
**Pull request by saumya665**

### CODE REVIEW

**Comment by MadhuBeesetty** [comment]
you need to add module.exports to test this file

File: addNum.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+const myfunc=(a,b)=>{
+  const c = a+b;
+  return c;
+};
+console.log(myfunc(2,3));
```
Comment (dsope05) [inline addNum.js]: need to export your function

Comment (dsope05) [reply addNum.js]: also add in the test for this file

File: addNum.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
```
Comment (songz) [inline addNum.js]: remove console.log. Your testing should console.log stuffs

**Pull request closed without merging**